src/find_library.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rcpputils/find_library.hpp"
+
+#include <cstddef>
+
+#include <list>
+#include <fstream>
+#include <sstream>
+
+namespace rcpputils
+{
+
+namespace
+{
+
+std::string get_env_var(const char * env_var)
```

Comment:
Er, actually - I wasn't sure why this Win32 code was there. I'm just using `getenv` instead. If there's a reason, I can add it back, but it should be documented because it's non-obvious.